lms/static/js/spec/groups/views/cohorts_spec.js

```diff
@@ -277,21 +281,23 @@ define(['backbone', 'jquery', 'js/common_helpers/ajax_helpers', 'js/common_helpe
                     expect(cohortsView.$('.cohort-management-nav')).toHaveClass('is-disabled');
                     expect(cohortsView.$('.cohort-management-group')).toHaveClass('is-hidden');
                     cohortsView.$('.cohort-name').val(defaultCohortName);
+                    cohortsView.$('.type-random').prop('checked', true).change();
```

Comment:
Do you also want to test not changing the assignment type (sticking with the default of manual)?